op-node/sources/rollupclient.go

```diff
@@ -0,0 +1,43 @@
+package sources
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-node/eth"
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+type RollupClient struct {
```

Comment:
Is this meant to be exhaustive in the RPCs it can call?

Reply:
those are in the p2p package in op node